src/sentry/lang/javascript/processing.py

```diff
@@ -175,6 +175,7 @@ def _handles_frame(frame, data):
         return False
 
     # Skip "native" frames
+    # TODO @athena: remove comment before merging
```

Comment:
I just assumed because this is in `javascript.processing` the new platform would not count as native in this context and didn't add it. Please correct me if I'm wrong.